test/index.js

```diff
@@ -132,6 +132,44 @@ describe('stand-in', function () {
     });
   });
 
+  describe('prototypes', function () {
+    it('replaces a prototype method', function (done) {
```

Comment:
These are basically the same as the tests a few lines up. How were they passing before but there is still an issue?